index.html

```diff
@@ -9,24 +9,71 @@
   <body>
     
      <div class="wrapper">
-      <h3>All Awesome Books</h3> 
-      <!---<div class="container-book"></div>--> 
-      <div class="book-from-storage"> </div> 
+        <header>
+          <nav class="menu-bar">
+            <li class="logo">
+              <a href="#" class="menu-item-link">
+                Awesome Books
+              </a>
+            </li>
+            <ul class="menus">
+              
+              <li class="menu-item">
+                <a href="#list" class="menu-item-link list-anchor">
+                  List
+                </a>
+              </li>
+              <li class="menu-item">
+                <a href="#add-new" class="menu-item-link add-new-anchor">
+                  Add new
+                </a>
+              </li>
+              <li class="menu-item">
+                <a href="#contact" class="menu-item-link contact-anchor">
+                  Contact
+                </a>
+              </li>           
 
-      <div class="hor-line"></div>
+            </ul>
+          </nav>
+        </header> 
+        <span class="time">March 19th 2021 9:25:32</span>
 
-      <h3>Add a new book</h3>
+        <section id="list" class="book-list-section">
+          <h3>All Awesome Books</h3> 
+        
+          <div class="book-from-storage"> </div> 
+        </section>
 
-      <div class="input-field">
-        <input type="text" placeholder="Title" class="title"/> 
-      <input type="text" placeholder="Author" class="author"/> 
-      </div>
-    
-      
-        <button type="button" class="add-btn">Add</button>
+        <section id="add-new" class="add-new-section hidden">
+          <h3>Add a new book</h3>
+
+          <div class="input-field">
+            <input type="text" placeholder="Title" class="title"/> 
+          <input type="text" placeholder="Author" class="author"/> 
+          </div>
+        
+          
+          <button type="button" class="add-btn">Add</button>
+        </section>
+
+        <section id="contact" class=" contact-section hidden">
+          <h2>Contact Information</h2>
+          <p>
+            Do you have any questions or you just want to say "Hello"? <br />
+            You can reach out to us!
+          </p>
+          <ul>
+            <li>Our email: amaremek@gmail.com</li>
+            <li>Our phone number: +2519 13 85 03 28</li>
+            <li>Our address: Streetname 22. 84503 Addis Ababa, Ehtiopia</li>
+          </ul>
+        </section>
+
+          <div class="copyright"><span>copyright</span></div>
```

Comment:
- [ ] Kindly give your footer the correct styling so that it matches the provided wireframe

**This is your design:**

![image](https://user-images.githubusercontent.com/86318284/193426912-268f6b77-994c-4678-8da1-5e03f5d192d7.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/193426927-bf00b40a-b019-450d-8b55-aabfb86f1197.png)

Reply:
Okay